src/pages/Admin/AdminMainLayout.tsx

```diff
@@ -4,9 +4,10 @@ import styled from 'styled-components';
 import AdminNavbar from './AdminNavbar';
 
 const Container = styled.div`
-  max-width: 1200px;
-  padding: 0 60px;
+  width: 100vw;
```

Comment:
I actually put 1200px max cause if it's 100% width, on large screen, it'll expand the the whole thing

Reply:
I tried adding max-width back in, but instead it was causing the container to only go to 1200px (the container beside the navbar is the bright purple in the screenshot). I left the max-width property out for now but if I am mistaken we can of course add it back in!
![container at max-width 1200](https://user-images.githubusercontent.com/87457132/167975598-43538317-8758-4ce0-af70-66161efde2bd.jpg)
.